yt/go/yt/interface.go

```diff
@@ -1501,6 +1510,47 @@ type GenerateTimestampOptions struct{}
 
 type GetInSyncReplicasOptions struct{}
 
+type AuthClient interface {
+
```

Comment:
Let's remove this empty line.

Usually we don't have those right in the beginning of any block (function body, interface, for loop body...)

Reply:
done